protocol/x/clob/e2e/long_term_orders_test.go

```diff
@@ -1404,3 +1405,401 @@ func TestPlaceLongTermOrder(t *testing.T) {
 		})
 	}
 }
+
+func TestRegression_InvalidTimeInForce(t *testing.T) {
```

Comment:
This test is exactly the same as one of the e2e tests above. The only difference is that instead of using a short term order as the maker, this test manually writes a stateful order with invalid TIF to state and use that as the taker.